js/notifications_ajax.js

```diff
@@ -0,0 +1,146 @@
+// Ajax notifications
+//
+// Inspired from BrowserNotification plugin by
+// Edgard Lorraine Messias
+//
+// License: BSD 3 clause
+
+(function (window, $) {
+   function GLPINotificationsAjax(options) {
+
+      var _this = this;
+      var _queue = $('<div />');
+      var _queue_audio = $('<div />');
+      var _interval = null;
+
+      this.options = $.extend({}, GLPINotificationsAjax.default, options);
+
+      var _this = this;
+
+      function showNotification(id, title, body, url) {
```

Comment:
a comment on the structure of the entire file.
function stored in var of GLPINotificationsAjax prototype should be better (mainly for scope, but also for consistensy)
like here: https://github.com/glpi-project/glpi/pull/2207/files?diff=unified#diff-87d084c06854757d1cee75d1d4d97fc5R85